frontend/src/audio.html

```diff
@@ -0,0 +1,271 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+jsFile: audio.js
+---
+<article
+  class="m-10 flex flex-col"
+  id="audio-page"
+  data-audio-number="{{ audio.objectID }}"
+  data-audio-title="{{ audio.title }}"
+  data-dateforhumans="{{ audio.dateForHumans }}"
+  data-dateuncertain="{{ audio.dateUncertain }}"
+  data-location="{{ audio.location }}"
+  data-locationuncertain="{{ audio.locationUncertain }}"
+  data-category="{{ audio.category }}"
+  data-languages="{{ audio.languages | json | escape }}"
+  data-duration="{{ audio.duration }}"
+>
+  <div class="flex flex-col">
+    <div class="flex justify-between align-baseline">
+      <!-- Title -->
+      <span class="text-lg font-bold">{{ audio.title }}</span>
+      <!-- Duration -->
+      <span>{{ audio.duration | duration }}</span>
+    </div>
+    <div class="mt-1 flex flex-col justify-between md:flex-row">
+      {% render 'partials/tagline.html', audio-id: audio.objectID, audio-date: audio.dateForHumans, audio-location: audio.location, audio-category: audio.category, audio-languages: audio.languages -%}
+      <div class="flex justify-end pt-1 md:pt-0">
+        <div class="flex">
+          <div class="flex self-stretch">
+            <span class="p-1">
+              <div
+                class="{{ audio.soundQualityRating | soundquality }} flex flex-none space-x-1 rounded-lg border px-1.5 text-xs"
```

Comment:
This `soundquality` filter may have more descriptive name, something like `sound_quality_color`.

Reply:
updated filter name